src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Table from 'paragon/src/Table';
+
+const AssetsTable = ({ assetsList }) => (
```

Comment:
FYI: this is a [stateless component](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md). When a component has no methods other than `render`, it's recommended to rewrite it as stateless for simplicity's sake.